coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/ArtifactsResolver.java

```diff
@@ -124,14 +124,8 @@ private ResolveReport getDependencies(DependencySpec dependencySpec) {
         .setOutputReport(true).setValidate(true).setTransitive(dependencySpec.transitive);
     resolveOptions.setLog(LogOptions.LOG_QUIET);
     try {
-      final ResolveReport report = _ivyInstance.resolve(md, resolveOptions);
-      if (report.hasError()) {
-        throw new RuntimeException("Unable to fetch dependencies: " + report.getAllProblemMessages());
```

Comment:
Is this the case that other exceptions will be wrongly captured here before?

Reply:
It might be this case, because after removing it, one affected view could be translated well even if there is still error in `report` like:
```
unresolved dependency: XXX: configuration not found in XXX: 'master'. It was required from YYY compile
```

And for most of the affected production views (8 in 10), after removing this exception and adding UDFs manually to avoid exceptions like `java.lang.RuntimeException: Unable to find/call org.apache.hadoop.hive.ql.udf.generic.GenericUDF.initialize() on: <UDF>`, there is still an exception: `java.lang.ArrayIndexOutOfBoundsException`, which is because Calcite doesn't support source SQL with "SELECT table.nested.field FROM (..)" logic. For more details, see: [CALCITE-1518](https://issues.apache.org/jira/browse/CALCITE-1518). So they still can't be translated well.

Reply:
I see. Basically `report.hasError()` can have false positive resolving exceptions. A followup question will be: if we remove it, will there be false negative (it is resolving exception but not get captured)?

Reply:
I tested this PR on **all** the production views.
Among the 32 affected views which failed because of `Unable to fetch dependencies: [XXX]`, 4 of them could be translated well with this PR, and 28 of them would throw known exceptions (most are `java.lang.ArrayIndexOutOfBoundsException`). Therefore, I think at least for all the existing views, there is no false negative case.